source/manual/connect-to-vcloud-director.html.md

```diff
@@ -4,39 +4,55 @@ title: Connect to vCloud Director (Carrenza only)
-You can use either Cisco AnyConnect or openconnect to do this.
+You can use either Cisco AnyConnect or OpenConnect as a VPN client for this.
 
-## Prerequisites
+## Setting up the Cisco AnyConnect VPN profile on a Mac
 
-1. Get the VPN client certificate from the [2nd line password store](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/carrenza/vpn-certificate.gpg).
-2. Save the certificate to a file on your machine (eg. `vcloud.pem`).
-3. Get the VPN credentials, also from the 2nd line password store (e.g.
-for integration this would be in [vcloud-integration.gpg](https://github.com/alphagov/govuk-secrets/tree/master/pass/2ndline/carrenza) ). You can find
-additional instructions about decrypting these files [here](https://github.com/alphagov/govuk-secrets/blob/master/pass/README.md) under the heading "Get a password".
-4. You'll need to use an app (such as Google Authenticator) to turn the TOTP
-   secret into a 2FA code.
-
-## Connecting with Cisco AnyConnect
-
-1. Convert the VPN client certificate from PEM format to PFX format by running
-   `openssl pkcs12 -export -in vcloud.pem -out vcloud.pfx`. You'll be asked for
-   two passwords. For the first one, enter the VPN password, and for the second
-   one, enter the certificate passphrase.
-2. Import the PFX format certificate into your Keychain by running
-   `security import vcloud.pfx -k ~/Library/Keychains/login.keychain-db`.
-   You'll be asked for a password. Enter the certificate passphrase.
-3. Create a new file on your machine at `/opt/cisco/anyconnect/profile/carrenza-secure.xml`.
+1. Clone the govuk-secrets private Git repo. The repo is very large but there is no need to download the full history.
+```
+mkdir -p ~/govuk && cd ~/govuk
+git clone --depth 10 git@github.com:alphagov/govuk-secrets.git
+```
+2. Get the VPN client certificate and private key from the 2nd line [password store](https://github.com/alphagov/govuk-secrets/tree/master/pass) and save the decrypted contents to a file on your machine (for example ~/carrenza-vpn-cert-and-key.pem).
+```
+cd govuk-secrets/pass
+export PASSWORD_STORE_DIR=2ndline
+pass carrenza/vpn-certificate >~/carrenza-vpn-cert-and-key.pem
+```
+3. Get the VPN credentials, also from the 2nd line password store.
+```
+pass carrenza/vpn-credentials
+Certificate passphrase: ...
+MFA key: ................
+Password: ...
+VPN gateway: ...
+```
+4. Add the MFA key to an app such as Google Authenticator. This will be your second factor for logging into the Carrenza VPN.
+5. Convert the VPN client certificate from PEM format to PFX format by running
+   `openssl pkcs12 -export -in ~/carrenza-vpn-cert-and-key.pem -out carrenza-vpn-cert-and-key.pfx`.
+   You'll be asked for two passwords (one for decrypting the PEM and one for encrypting the PFX). The first one is the `Certificate passphrase` field from `carrenza/vpn-credentials`. You can choose the second one, or use the same as the first.
```

Comment:
Is there a better way of saying `You can choose the second one, or use the same as the first.`? Found it confusing at first

Reply:
Good catch! Reworded this - hopefully makes more sense now.